dts/bindings/video/st,stm32-dcmi.yaml

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+    STMicroelectronics STM32 Digital Camera Memory Interface (DCMI)
+    example:
+
+    &dcmi {
+      status = "okay";
+      sensor = <&ov2640>;
```

Comment:
Shouldn't this also mention `compatible= ...`?

Reply:
Do I need to repeat here? Because it was declared into the dcmi node on `dts/arm/st/h7/stm32h743.dtsi`. In the sample code https://github.com/zephyrproject-rtos/zephyr/pull/71463 the build process didn't complain. Please, let me know if a need to repeat.

Reply:
Agreed compatible is not required because we're documenting example of board/app configuration.
Maybe this could be made explicit to avoid confusion: "Example of node configuration at board level"

Reply:
Done